graspy/match/gmp.py

```diff
@@ -226,7 +220,7 @@ def fit(self, A, B, seeds_A=[], seeds_B=[]):
         B21T = np.transpose(B21)
         B22T = np.transpose(B22)
 
-        for i in range(self.n_init):
+        def _fit_single_init(init_num):
```

Comment:
init_num is not used, correct? if so, remove as arg and don't pass anything to this function